File: backend spring/src/main/java/com/bloodbank/dto/response/DonationRecordDto.java
package com.bloodbank.dto.response;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;
import java.time.LocalDateTime;

@Getter
@Setter
public class DonationRecordDto {
    private Long recordId;
    private DonorSimpleDto donor;
    private LocalDate donationDate;
    private int unitsDonated;
    private LocalDateTime createdAt;
}

File: BackendDotNet/DTOs/Response/DonationRecordDto.cs
namespace BackendDotNet.DTOs.Response
{
    public class DonationRecordDto
    {
        public long RecordId { get; set; }
        public DonorSimpleDto Donor { get; set; } = new();
        public DateTime DonationDate { get; set; }
        public int UnitsDonated { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}
